conf/messages

```diff
@@ -126,7 +126,7 @@ gmp.month = Month
 gmp.name = Name
 gmp.nino = National Insurance number
 gmp.nino.hint = For example, QQ123456D
-gmp.scon = Scheme Contracted Out Number (SCON)
+gmp.scon = Enter Scheme Contracted Out Number (SCON)
```

Comment:
Is this the right value to change? The ticket says to change the H1 which would be message `gmp.pension_details.header`